Documentation/install/aws/aws-terraform.md

```diff
@@ -111,6 +111,20 @@ $ cp examples/terraform.tfvars.aws build/${CLUSTER}/terraform.tfvars
 
 Edit the parameters with your AWS details, domain name, license, etc. [View all of the AWS specific options][aws-vars] and [the common Tectonic variables][vars].
 
+### Set Console login secrets
+
+Set these sensitive values in the environment. Terraform will encrypt them before storage or transport:
+
+* `TF_VAR_tectonic_admin_email` - String giving the email address used as user name for the initial Console login
```

Comment:
`tectonic_admin_email` is technically not a "sensitive" value. It's something that could be stored in the environment, but doesn't really need to be.

`tectonic_admin_password` however, is. The password var is bcrypt-ed in the installer, but the email var is not.

Same comment for each of the files.

Reply:
my thinking is that these are thought of together, so they should be set together

Reply:
I don't disagree that they [c|sh]ould be set together, just noting that while the pw is encrypted, the email is not.

Reply:
@robszumski - @justaugustus and I spoke about this and agreed that both values are "sensitive," but that we could be more specific about what is encrypted. Changes pushed to that effect.